README.rst

```diff
@@ -55,11 +55,10 @@ solutions. The hope is for Matrix to act as the building blocks for a new
 generation of fully open and interoperable messaging and VoIP apps for the
 internet.
 
-Synapse is a reference "homeserver" implementation of Matrix from the core
-development team at matrix.org, written in Python/Twisted.  It is intended to
-showcase the concept of Matrix and let folks see the spec in the context of a
-codebase and let you run your own homeserver and generally help bootstrap the
-ecosystem.
+Synapse is a Matrix "homeserver" implementation developed by the matrix.org core 
+team, written in Python 3/Twisted. It is intended to showcase the concept of Matrix 
+and let folks see the spec in the context of a codebase and let you run your own 
+homeserver and generally help bootstrap the ecosystem.
```

Comment:
I don't think this second sentence is true any more, either :/

```suggestion
team, written in Python 3/Twisted.
```

Reply:
Changed that!